includes/class-newspack-popups-model.php

```diff
@@ -1100,6 +1100,9 @@ class="<?php echo esc_attr( implode( ' ', $classes ) ); ?>"
 			id="<?php echo esc_attr( $element_id ); ?>"
 			data-segments="<?php echo esc_attr( $assigned_segments ); ?>"
 			data-frequency="<?php echo esc_attr( $frequency_config ); ?>"
+			<?php if ( ! empty( $utm_suppression ) ) : ?>
```

Comment:
`$utm_suppression` needs to be defined before this:

```php
$utm_suppression = $popup['options']['utm_suppression'];
```

Reply:
Whoops! 16593339fc10b44868926ec8b9c1d80f0fdf0d83